### DISCUSSION TITLE
各位觉得那个正版登录图标好看？

### DISCUSSION THREAD

**Discussion by ZerkyLiu** [Meloong-Git/PCL#3983] [讨论]
感觉PCL2的旧正版登录图标比现在的更好看。现在的图标有一股杀毒软件的味道
旧的：
![image](https://github.com/Hex-Dragon/PCL2/assets/115980764/f3a18bdc-8f54-4b1d-8436-9154997cb620)

新的：
![image](https://github.com/Hex-Dragon/PCL2/assets/115980764/eb4339d3-d7e7-40fa-80bd-22b347693d12)

**Comment by SnifferGX** [comment]
勾号我个人认为有Mojang认证的感觉，旧版的太文字化了

**Comment by MoYuan-CN** [comment]
~~我怎么没见过第一张图的 Icon~~

**Comment by wuliaodexiaoluo** [comment]
我觉得新版好看

**Comment by iPodtouch6557** [comment]
新版比旧版好，但是新版其实也不好

**Comment by 3gf8jv4dv** [comment]
Maybe consider using Microsoft’s brand icon? 

- Minecraft is now owned by Microsoft, and the current official account system has been transformed to use Microsoft accounts
- There are already other launchers that do this (e.g. HMCL)

**Reply by zkitefly** [reply]
以前使用过 Windows 10 的图标，但不知道什么时候就换成这个了

**Reply by iPodtouch6557** [reply]
> 以前使用过 Windows 10 的图标，但不知道什么时候就换成这个了

Windows11的图标似乎可以
或者win7甚至win95也行
